clusterDeployer.py

```diff
@@ -228,18 +228,31 @@ def ensure_linked_to_bridge(self):
             sys.exit(-1)
 
     def deploy(self):
-      if not self.args.onlypost:
-        if not self.args.skipmasters and self._cc["masters"]:
+      steps = self.args.steps.split(",")
+      if self._cc["masters"]:
+        if "pre" in steps:
           self._preconfig()
-          self.teardown()
-          self.create_cluster()
+        else:
+          print("Skipping pre configuration.")
+
+        self.teardown()
```

Comment:
Should we add a skip for this?

Reply:
no, I don't want the user to skip teardown manually. We can have logic that detects when to run teardown, but it should not be something explicitly skipped by the user at this point. We already have -t

Reply:
Hmm, wondering if we should skip setting up the cluster then? If there are cases were we want to just test pre/post config.

Reply:
Not in this PR, but we can support that too in the future